simp-beaker-helpers.gemspec

```diff
@@ -27,6 +27,11 @@ Gem::Specification.new do |s|
   # TODO: Update this when we no longer support Ruby 1.9 (should be October 2018)
   s.add_runtime_dependency 'fog-openstack', '0.1.25'
 
+  # Because net-telnet dropped support for Ruby < 2.3.0
+  # TODO: Update this when we no longer support Ruby 1.9 (should be October 2018)
+  s.add_runtime_dependency 'net-telnet', '~> 0.1.1'
```

Comment:
Do you mean ruby 2.1.9? and can this be wrapped in a ruby version check so I can still get the latest on rubies > 2.1.9?

Reply:
@jeefberkey No, it can't because of how `gemspec` files work.

Reply:
@jeefberkey Not in a `.gemspec`—if you put in a ruby version check like that, gems would only reflect the version of ruby that the build host was running at the time they were packaged.  

You can put conditional requirements like that in a `Gemfile`, but that's something that each downstream project would have to configure for themselves.